### PR TITLE
Add Request::get_file_mutil_value interface.

### PR DESCRIPTION
**Pull request by jingTian-z**
Return multiple values corresponding to the key.

### CODE REVIEW

**Comment by yhirose** [comment]
@jingTian-z, thanks for the contribution. Could you provide a Unit test for the method?

**Comment by yhirose** [comment]
@jingTian-z, ping?

**Comment by jingTian-z** [comment]
I am very sorry to see your email now. I am taking a holiday now but after my vacation, I think I can have a try.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: yhirose ***@***.***>
发送时间: Sunday, May 2, 2021 8:29:06 PM
收件人: yhirose/cpp-httplib ***@***.***>
抄送: jingTian-z ***@***.***>; Mention ***@***.***>
主题: Re: [yhirose/cpp-httplib] Add Request::get_file_mutil_value interface. (#918)


@jingTian-z<https://github.com/jingTian-z>, ping?

―

File: httplib.h [hunk 1]
```diff
@@ -3833,6 +3835,18 @@ inline MultipartFormData Request::get_file_value(const char *key) const {
   return MultipartFormData();
 }
 
+/**
+ * Return multiple values corresponding to the key
+*/
+std::vector<MultipartFormData> Request::get_file_mutil_value(const char *key) const {
```
Comment (yhirose) [inline httplib.h:3841]: Could you add `inline` keyword at the beginning of the line as other methods do?

**Review by yhirose — CHANGES_REQUESTED** [review]

**Comment by yhirose** [comment]
I'll close it for now until the code gets revised.

**Pull request closed without merging**